recipes/semver200/all/test_package/CMakeLists.txt

```
@@ -0,0 +1,11 @@
+cmake_minimum_required(VERSION 3.11)
+project(test_package CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
```

```suggestion
conan_basic_setup(TARGETS)
```
So cmake generator will not mask cmake_find_package